README.md

```diff
@@ -155,6 +156,7 @@ The Friendica image supports auto configuration via environment variables.
 You can preconfigure everything that is asked on the install page on first run.
 To enable the automatic installation, you have to the following environment variables:
 
+-	`FRIENDICA_URL` The Friendica URL.
```

Comment:
This should be more specific about what's expected. Is it scheme + domain + eventual root path of the intended new Friendica node? If so it should be explicit, either in the description or in a sample URL.